Project: Apple  Clone

OlukaDenis opened the pull request:
Features:
1. Applied the technique of linear gradient to the navigation bar.
2. Applied a background with a picture
3. Using display and position for laying out items

OlukaDenis committed:
Layout building:
1. Created the layout containers
2. Added the background picture
3. Added the middle section contents
4. Added the navigation

OlukaDenis committed:
Header styling:
1. Style all headers in the main content section

OlukaDenis committed:
Added and styled the footer contents

OlukaDenis committed:
Renamed the stickler.yml to .stickler.yml

OlukaDenis committed:
Resolved the stickler error

OlukaDenis committed:
Merge pull request #2 from OlukaDenis/appleclone-footer

Appleclone footer

stickler-ci committed:
Fixing style errors.

OlukaDenis committed:
Solved the stickler errors

stickler-ci committed:
Fixing style errors.

bolah2009 commented on index.html:
```diff
@@ -0,0 +1,133 @@
+          <li class="nav-item"><span>iPhone</span></li>
+          <li class="nav-item"><span>iPad</span></li>
+          <li class="nav-item"><span>iTunes</span></li>
+          <li class="nav-item"><span>Support</span></li>
```
I think it would be better to use the `button` element here rather than `span` as `button` element give semantic meaning to the buttons compared with the `span` element.

bolah2009 commented on style.css:
```diff
@@ -0,0 +1,317 @@
+}
+
+.asidebar ul {
+  display: flex;
```
Let's add `width` property to the `li.aside-item` and `li.aside-item img` elements here.

The images are taking more space than they should.

![apple-foot-2](https://user-images.githubusercontent.com/36057474/65810826-9a897b00-e1a7-11e9-8cbd-469786c3dd48.gif)

bolah2009 reviewed (changes requested):
# Nice job @OlukaDenis 

### [The clone](https://rawcdn.githack.com/OlukaDenis/Apple-Clone/efb26a76df1f39508f87220c8a68bacaddb55d4a/index.html) looks good to me and uses CSS Backgrounds and gradients effectively. :100: 

A few required changes :relaxed: 
- [REQUIRED] Add a descriptive readme. Details like assignment link (from Odin), live link (from [githack](https://raw.githack.com/)), screenshot and what was learned during the project can make the project more descriptive.
  > Remember about descriptive README file.
- [REQUIRED] The promotion images align as illustrated in the comment below
- Let's make this Pull Request look more professional.
  - [NEXT PROJECT]: Use meaningful commits messages. [Learn more here](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8/) and Check [this awesome article](https://chris.beams.io/posts/git-commit/)
  - [:heavy_check_mark:] Use meaningful names for  Pull Requests.
  - [:heavy_check_mark:] Add a short summary of changes in Pull Request description.

#### :recycle:  Make these changes and submit another code review request. :+1: 

#### Feel free to reach me on slack if you having issues implementing these features. :+1: 

Good luck with your next submission. :star2: 

Cheers
[Bola](https://bolabuari.com/)

###### PS: Note that comments tagged as `OPTIONAL` are not part of the minimum requirements to approve this PR

OlukaDenis committed:
PR changes:
1. Added the screenshot
2. Fixed the bottom asidebar width issue
3. Updated the README file

OlukaDenis committed:
Merge branch 'appleclone-layout' of github.com:OlukaDenis/Apple-Clone into appleclone-layout

OlukaDenis committed:
Fixed a stickler error

stickler-ci committed:
Fixing style errors.

stickler-ci committed:
Fixing style errors.

ebukaume commented:
Hi @OlukaDenis :wave: 

Nice job so far. Your README now looks good but there is one more requirement we need to address.

**Bottom**

- [ ] It contains four elements **which are links** (to other articles or #)

[OPTIONAL]
Your page falls apart when I zoom out to scales below 100%. It would be wonderful if you could fix that too. :wink: 

**Please submit another code review request form as soon as you fix the issue.** :v:

OlukaDenis committed:
Fixed the aspect ratio issue

OlukaDenis committed:
Merge branch 'appleclone-layout' of github.com:OlukaDenis/Apple-Clone into appleclone-layout

OlukaDenis committed:
Updated the live link

bolah2009 reviewed (approved):
# Nice job implementing the suggested changes. 

### It's a :+1: from me.

You can merge to master.
